clox/functions.py

```diff
@@ -231,6 +231,12 @@ def run_clock(
         if not hide_timezone:
             print(" " * h_shift, end='')
             print("Timezone: {timezone}".format(timezone=timezone_str))
+            if tz is not None:
+                datetime_now_local = datetime.datetime.now()
+                time_formats_local = HORIZONTAL_TIME_12H_FORMATS if am_pm else HORIZONTAL_TIME_24H_FORMATS
+                current_time_local = datetime_now_local.strftime(time_formats_local[0])
```

Comment:
Shouldn't we use the same `format_index` here as well, instead of `0`?

Reply:
fixed in https://github.com/sepandhaghighi/clox/pull/54/commits/41a8229a1df92c04f7bb44e795fdf83c6f2086a5